packages/flutter/lib/src/gestures/force_press.dart

```diff
@@ -190,7 +189,7 @@ class ForcePressGestureRecognizer extends OneSequenceGestureRecognizer {
   ///
   /// This function must always return values in the range 0.0 to 1.0 given a
   /// pressure that is between the minimum and maximum pressures. It may return
-  /// [double.NaN] for values that it does not want to support.
+  /// `double.NaN` for values that it does not want to support.
```

Comment:
I wonder if it'd link it this was `[dart:core.double.NaN]`?

Reply:
No luck.  I think this is probably a dartdoc bug but I'm also not sure how important it is we link to `double.NaN` here.